frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -87,6 +100,14 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
   const [guestAgentInfoRaw] = useGuestAgentInfo({ vmi });
   const guestAgentInfo = new GuestAgentInfoWrapper(guestAgentInfoRaw);
   const operatingSystemID = guestAgentInfo.getOSInfo().getId();
+  const [dvs, setDataVolumes] = React.useState(null);
+  const loadedDataVolumes = getLoadedData(props.dataVolumes);
+
+  React.useEffect(() => {
```

Comment:
what is this for?

Reply:
There is a strange behavior when fetching DataVolumes from firehose which it pulls the DVs from the cluster OK at the first but at the end it returns null, so your are stuck with null  instead of a list of DVs - this is a workaround for keeping the DVs you got.

Reply:
Can you debug this please more? Why doesn't it happen in other places we use datavolumes?

Is this a problem/bug with firehose or the env? Do periodic GET requests cause this bug as well?

Reply:
I can't reproduce it for some reason. I'll remove it.